lib/helpers.js

```diff
@@ -33,3 +34,20 @@ export async function getZcapClient({config} = {}) {
 
   return {zcapClient, zcaps};
 }
+
+export function sendCacheableJson({
+  res, obj, maxAge = 300, sMaxAge = undefined
```

Comment:
Note: I imagine if we extract this out into a reusable library we'd pass things like `maxAge` as cache options (namespace them instead of passing as top-level options with `res` and `obj`).